python/interpret_text/common/dataset/utils_mlni.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
```

Comment:
Can we move all of the utils outside of interpret_text. Possibly into the directory of the notebook for now. Longer term we will have to see how best to minimize our maintenance costs for these tools.

Are there internal interpret_text dependencies on the utils? If so, please @ me for those. If not, moving these to notebooks for now should allow us to move forward faster